src/components/_common/icons/SwitchThemeIcon.tsx

```diff
@@ -0,0 +1,31 @@
+import { IconProps } from 'src/components/_common/icons/types';
+
+export const SwitchThemeIcon = ({ color = 'inherit', size = 'inherit' }: IconProps) => (
+  <svg
+    width="50"
+    height="50"
+    viewBox="0 0 28 28"
+    style={{ fontSize: size, color, width: '1em', height: 'auto' }}
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <circle cx="14.3923" cy="14.3923" r="6" fill="#CDCDCD" />
+    <path d="M16.3923 6.39233H12.3923L14.3923 2.39233L16.3923 6.39233Z" fill="#D9D9D9" />
```

Comment:
fill = background-color for SVG syntax. Seems like applied color should do nothing

Reply:
done